de.properties.tmpl

```diff
@@ -0,0 +1,201 @@
+{{- with $base := (printf "configs/%s" (env "DE_ENV")) -}}
+
+
+###############################################################################
+# Deployed Version Metadata
+#
+# This values are used since reading the MANIFEST.MF file from a deployed WAR
+# is a tricky environment.
+###############################################################################
+org.iplantc.discoveryenvironment.about.defaultBuildNumber = {{ key (printf "%s/de/default-build-number" $base) }}
+org.iplantc.discoveryenvironment.about.releaseVersion = {{ key (printf "%s/de/release-version" $base) }}
+
+###############################################################################
+# CAS Authentication Settings
+###############################################################################
+org.iplantc.discoveryenvironment.cas.base-url = {{ key (printf "%s/de/base-url" $base) }}
+org.iplantc.discoveryenvironment.cas.server-name = {{ key (printf "%s/de/cas-server-name" $base) }}
```

Comment:
I think this one might be the one that could be changed to `/de/base` or something like that? we looked at it briefly while trying to fix the JSON stuff yesterday